starter/js/atm.js

```diff
@@ -1,15 +1,53 @@
 //Begin with the document ready function
+$( document ).ready(function() {
+});
+
+
+// START BELOW
+
+
+
+    var checkingBalance = 0;
+    var savingsBalance = 0;
 
     //Checking account deposit function
+    var depositChecking = function() {
+      var amount = parseInt(document.querySelector('#amountChecking').value);
+      document.querySelector('#amountChecking').value = "";
+
```

Comment:
For some reason my jQuery wasn't applying. Made sure it was correct, but it wasn't happening so I just kept doing it in vanilla JS. 



> On Jun 10, 2016, at 12:32 PM, Amanda Chang <notifications@github.com> wrote:
> 
> In starter/js/atm.js:
> 
> >  
> >      //Checking account deposit function
> > +    var depositChecking = function() {
> > +      var amount = parseInt(document.querySelector('#amountChecking').value);
> > +      document.querySelector('#amountChecking').value = "";
> > +
> Interesting decision to go with vanilla JS and no jQuery! Was this an intentional choice?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>